src/components/Sidebar.tsx

```diff
@@ -72,7 +73,9 @@ const Sidebar = (props: SideBarProps) => {
               previousMarkers: previousMarkers,
               clearMarkers: clearMarkers,
             }}
+            isFavorited={props.favs.map((fav) => fav.id).includes(otherUser.id)}
```

Comment:
Wouldn't it be better to just create a const which is a Set of the favorited ids? That way we would only loop through props.favs to create the set, then access time is O(1). Currently, we are looping through favs for every recommended user, when we could just loop through once. Unless I am misunderstanding something

Reply:
good point